andela/goodfellas-ah-backend
# #160618090 Configure PostgreSQL as DB Engine

### What does this PR do?
This pull request removes mongodb and integrates postgreSQL

### Description of Task to be completed?
- Remove mongodb and its dependencies
- Install and configure postgres
- Install and configure sequelize
- Add .env  and .env.example file
- Create a test model to check database connection

### How should this be manually tested?
- After cloning the repo, cd into it
- Install depencies with `npm install`
- Create a .env file using the .env.example format
- Create a local postgres database and fill in the credentials in the .env file
- On the terminal run `sequelize db:migrate` to create a test model called article
- Run node index.js to start the server
- visit `localhost:3200/article` which should return a json with a success message




### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
#160618090

### Screenshots (if appropriate)
N/A

### Questions:
None
